include/mbedtls/mbedtls_config.h

```diff
@@ -3839,6 +3859,19 @@
  */
 //#define MBEDTLS_PSA_KEY_SLOT_COUNT 32
 
+/**
+ * \def MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE
+ *
+ * Define the size (in bytes) of each static key slot when
+ * MBEDTLS_PSA_STATIC_KEY_SLOTS is set. If not
+ * explicitly defined then it's automatically set to hold the maximum
+ * asymmetric PSA key enabled in the build (through PSA_WANT_xxx symbols).
```

Comment:
Sorry, symmetric keys were not listed here because they were not used in the first implementation. Then when I changed the code, I forgot to update this part.
As for the content of this paragraph and the "promises" it makes, what if I rephrase it as follows:
```
 * Define the size (in bytes) of each static key buffer when
 * #MBEDTLS_PSA_STATIC_KEY_SLOTS is set. If not
 * explicitly defined then it's automatically guessed from available PSA keys
 * enabled in the build through PSA_WANT_xxx symbols.
 ```
To me it looks more generic because we just mention generic "PSA keys" without specifying the type. Is that enough?